client/src/components/graph/overlays/centroidLabels.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable jsx-a11y/mouse-events-have-key-events */
+import React, { PureComponent } from "react";
+
```

Comment:
Suggest adding a few comments describing what coordinate system his component is operating it, for future work updating fonts, etc.

Reply:
I added a comment in the `GraphOverlayLayer` explaining assumed origin coordinate space and the resulting coordinate space.